### PR TITLE
documenting MATLAB build options

### PR DESCRIPTION
**Pull request by mw-hrastega**

### CODE REVIEW

**Review by mw-kapilg — COMMENTED** [review]
I see we've used "box" at all places where we require a string input from the user but I relate more with "field". GPT says the following in reference to UI elements:

- Text Box: Often used for multiline input, such as descriptions, messages, or comments.
- Text Field: Typically used for single-line input, such as names, email addresses, or search terms.

File: CONFIGDOC.md [hunk 1]
```diff
@@ -48,20 +49,25 @@ The plugin provides you with three tasks:
...
 
-To configure the **Run MATLAB Build** task, first specify the MATLAB executable and optional startup options to use for the task. Then, specify the MATLAB build tasks you want to execute in the **Build tasks** box. If you specify more than one task, use a space to separate them. If you do not specify any tasks, the plugin runs the default tasks in `buildfile.m` as well as all the tasks on which they depend. For example, use MATLAB R2023b to run a task named `mytask` as well as all the tasks on which it depends.
+To configure the **Run MATLAB Build** task, first specify the MATLAB executable and optional startup options to use for the task. Then, specify your MATLAB build tasks and build options. If you specify more than one task in the **Tasks** box, use a space to separate them. If you do not specify any tasks, the plugin runs the default tasks in `buildfile.m` as well as all the tasks on which they depend. For example, use MATLAB R2023b to run a task named `mytask` as well as all the tasks on which it depends.
```
Comment (mw-kapilg) [inline CONFIGDOC.md:59]: Does "build" appear repetitive in "your MATLAB build tasks and build options"? How about "your MATLAB build tasks and options"?

File: CONFIGDOC.md [hunk 2]
```diff
@@ -48,20 +49,25 @@ The plugin provides you with three tasks:
...
 
-![run_matlab_build](https://github.com/mw-hrastega/Times-Table-App/assets/48831250/de110406-170c-4e10-a3ec-269862cdb8eb)
+<img width="full" alt="run_matlab_build" src="https://github.com/mathworks/matlab-bamboo-plugin/assets/48831250/97a03376-4c15-4ddb-af17-2a0c3ff133b1">
 
 MATLAB exits with exit code 0 if the build runs successfully. Otherwise, MATLAB terminates with a nonzero exit code, which causes the Bamboo build to fail.
 
-When you use this task, a file named `buildfile.m` must be in the root of your repository. For more information about the build tool, see [Create and Run Tasks Using Build Tool](https://www.mathworks.com/help/matlab/matlab_prog/create-and-run-tasks-using-build-tool.html).
```
Comment (sameagen-MW) [inline CONFIGDOC.md:64]: Did this line get moved somewhere else?

File: CONFIGDOC.md [hunk 3]
```diff
@@ -48,20 +49,25 @@ The plugin provides you with three tasks:
...
+#### Specify Build Options
+To specify build options for your MATLAB build, first select **Specify build options**  and then populate the box that appears in the task configuration interface. For example, specify `-continueOnFailure` to continue running the MATLAB build upon a build environment setup or task failure. The plugin supports the same [options](https://www.mathworks.com/help/matlab/ref/buildtool.html#mw_50c0f35e-93df-4579-963d-f59f2fba1dba) that you can pass to the `buildtool` command when running a MATLAB build.
```
Comment (sameagen-MW) [inline CONFIGDOC.md:66]: The "...when running a MATLAB build." part of this sentence seems to not add much value to me. I think we also removed it from the Jenkins plugin description?

File: CONFIGDOC.md (hunk 2 above)
Comment (mw-hrastega) [reply CONFIGDOC.md:64]: No, I decided to remove it. The first paragraph of the section covers this last paragraph.

File: CONFIGDOC.md (hunk 3 above)
Comment (mw-hrastega) [reply CONFIGDOC.md:66]: Fair enough! Removed it.

File: CONFIGDOC.md (hunk 1 above)
Comment (mw-hrastega) [reply CONFIGDOC.md:59]: Makes sense. Updated.

**Comment by mw-hrastega** [comment]
> I see we've used "box" at all places where we require a string input from the user but I relate more with "field". GPT says the following in reference to UI elements:
> 
> * Text Box: Often used for multiline input, such as descriptions, messages, or comments.
> * Text Field: Typically used for single-line input, such as names, email addresses, or search terms.

Unfortunately, we cannot use "field". Please refer to the last bullet on page 6 of our User Interface Elements guidelines: https://mathworks.sharepoint.com/:w:/r/sites/doc/kb/_layouts/15/Doc.aspx?sourcedoc=%7BD65C7A22-580A-4080-990C-2012EE9987F7%7D&file=User%20Interface%20Elements.docx&action=default&mobileredirect=true&DefaultItemOpen=1

File: CONFIGDOC.md [hunk 4]
```diff
@@ -48,20 +48,27 @@ The plugin provides you with three tasks:
...
+You can specify optional startup options for a MATLAB executable by first selecting **Specify startup options**  and then populating the box that appears in the task configuration interface. For example, specify `-nojvm` to start MATLAB without the JVM&trade; software. If you specify more than one startup option, use a space to separate them (for example, `-nojvm -logfile "output.log"`). For more information about MATLAB startup options, see [Commonly Used Startup Options](https://www.mathworks.com/help/matlab/matlab_env/commonly-used-startup-options.html).
 
 > :information_source: **Note:** Using the **Options** box to specify the `-batch` or `-r` option is not supported.
 
+> :information_source: **Note:** By default, when you use the **Run MATLAB Build**, **Run MATLAB Tests**, or **Run MATLAB Command** task, the root of your repository serves as the MATLAB startup folder. To run your MATLAB code using a different folder, include the `-sd` startup option or the `cd` command in the task.
```
Comment (mw-hrastega) [inline CONFIGDOC.md:55]: @sameagen-MW: Please pay attention to this new note and its placement.

**Review by mw-kapilg — APPROVED** [review]
Looking good :shipit:

**Pull request merged**